website/pages/docs/nia/installation/requirements.mdx

```diff
@@ -3,16 +3,60 @@ layout: docs
 ---
 
 # Prerequisites Needed to Run Consul-Terraform-Sync
 
-Consul-Terraform-Sync requires a Terraform Provider, a Terraform Module and a running Consul Cluster outside of the consul-terraform-sync daemon. Practitioners can add support for their network infrastructure through Terraform providers. Once network infrastructure support exists, practitioners can add network integrations in the form of Terraform modules.
+Consul-Terraform-Sync requires a Terraform Provider, a Terraform Module, and a running Consul cluster outside of the consul-terraform-sync daemon. Practitioners can add support for their network infrastructure through Terraform providers. Once network infrastructure support exists, practitioners can add network integrations in the form of Terraform modules.
+
+
+## Run a Consul Cluster
+
+Below are several steps towards a minimum Consul setup required for running Consul-Terraform-Sync.
+
+### Install Consul
+
+Consul-Terraform-Sync is a daemon that runs alongside Consul, similar to other Consul ecosystem tools like Consul Template. Consul-Terraform-Sync is not included with the Consul binary and needs to be installed separately.
+
+To install a local Consul agent, refer to the [Getting Started: Install Consul Tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-install).
+
+### Run an Agent
+
+The Consul agent must be running in order to dynamically update network devices. To run the local Consul agent, you can run Consul in development mode which can be started with `consul agent -dev` for simplicity. For more details on running Consul agent, refer to the [Getting Started: Run the Consul Agent Tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-agent?in=consul/getting-started).
+
+### Register Services
+
+Consul-Terraform-Sync monitors Consul catalog for service changes which lead to downstream changes to your network devices. Without services, your Consul-Terraform-Sync daemon will be operational but idle. You can register services with your Consul agent either by loading a service definition or by http api request.
+
+If you are running Consul in development mode, below is an example of registering a service by HTTP API request:
+
+```shell-session
+$ echo '{
+    "ID": "web",
+    "Name": "web",
+    "Address": "10.10.10.10",
+    "Port": 8000
+}' > payload.json
+
+$ curl --request PUT --data @payload.json http://localhost:8500/v1/agent/service/register
+```
+
+The above example registers a service named "web". You can optionally configure the "web" service with a [service block](/docs/nia/installation/configuration#service) if it has any non-default values. You can also have Consul-Terraform-Sync monitor the "web" service to execute a task and update network device(s) by configuring "web" in [`task.services`](https://deploy-preview-8881--consul-docs-preview.netlify.app/docs/nia/installation/configuration#services) of a task block.
```

Comment:
Could you clarify where the service is registered to and what you're configuring "web" for? It's ambiguous when switching context between registering to Consul agent to then configuring for Sync.

For example adding:

> The above example registers a service named "web"

... to the Consul agent.

Reply:
Ah, I see what you mean about it being ambiguous. What do you think of something like:

The above example registers a service named "web" with your Consul agent. This represents a non-existent web service running at 10.10.10.10:8000. Your web service is now available for Consul-Terraform-Sync to consume. In Consul-Terraform-Sync, you can optionally configure the web service with a [service block](/docs/nia/installation/configuration#service) if it has any non-default values. You can also have Consul-Terraform-Sync monitor the web service to execute a task and update network device(s) by configuring "web" in [`task.services`](/docs/nia/installation/configuration#services) of a task block.

Reply:
✨🛳️

yeah that's really good! i like the added context. it walks readers through from Consul to Sync

Reply:
Whoops! Your comment made me realize I also included the netlify link in the docs 🤦

Reply:
LOL i missed that. nice catch

Reply:
thanks @findkim 🙏